airbyte-integrations/connectors/source-kyriba/source_kyriba/source.py

```diff
@@ -270,9 +270,15 @@ def gateway_url(self, config: Mapping[str, Any]) -> str:
         return f"https://{config['domain']}/gateway"
 
     def check_connection(self, logger, config) -> Tuple[bool, any]:
-        client = KyribaClient(config["username"], config["password"], self.gateway_url(config))
-        client.login()
-        return True, None
+        try:
+            client = KyribaClient(config["username"], config["password"], self.gateway_url(config))
```

Comment:
Kyriba uses Oauth2 Protocol for authentication (access token will expire in 1 hour).
Do we need to migrate it as well?

Reply:
@artem1205 This is a community connector with few active users -- is the current authentication method inadequate?

Reply:
>  is the current authentication method inadequate?

Well, it does work but the token we obtain in `login` method will work only during **_1 hour_**. After that we need to exchange it to the new one.

Reply:
agreed, this will work